azulzulu-11/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM azul/zulu-openjdk:11.0.1-11.2
```

Comment:
this would be better `azul/zulu-openjdk:11` so it gets automatically updated when new images are published